src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -162,6 +163,12 @@ public async Task<ListResponseModel<OrganizationUserUserMiniDetailsResponseModel
     [HttpGet("")]
     public async Task<ListResponseModel<OrganizationUserUserDetailsResponseModel>> Get(Guid orgId, bool includeGroups = false, bool includeCollections = false)
     {
+
+        if (!_featureService.IsEnabled(FeatureFlagKeys.AccountDeprovisioning))
```

Comment:
I think this is wrong
```suggestion
        if (_featureService.IsEnabled(FeatureFlagKeys.SeparateCustomRolePermissions))
```